app/javascript/components/donations/Form.tsx

```diff
@@ -126,7 +128,7 @@ export const Form = ({
         switchToTab: (id) => setTransactionType(id as PaymentIntentType),
       }}
     >
-      <div className="c-donations-form">
+      <div id={id} className="c-donations-form">
```

Comment:
You'll also need to update https://github.com/exercism/website/blob/main/app/javascript/components/donations/FormWithModal.tsx#L39 to also pass in (a different) id.

Reply:
FYI The ID's need to be different as the two forms are shown on the same page on https://exercism.org/donate (one at the top, the other at the bottom)